[DOCS-7405] Update Flex Logs doc (#23542)
DataDog/documentation: content/en/logs/log_configuration/flex_logs.md

  text: "Log Archives"
---

## Overview

> As your organization scales, the volume of logs collected from your infrastructure and applications grows along with it. The use cases for your logs increase in complexity as well. For example, you might be collecting logs from your infrastructure, applications, security tools, network, and so forth.  All of these use cases have varying retention and querying needs.

Author: ```suggestion
As your organization scales, the volume of logs collected from your infrastructure and applications grows along with it. The use cases for your logs increase in complexity as well. For example, you might be collecting logs from your infrastructure, applications, security tools, network, and so forth. All of these use cases have varying retention and querying needs.
```

- Concurrency: The number of users concurrently querying Flex Logs.

Consider the following factors when deciding on a compute tier:

- Your daily log volume and the number of logs stored in the Flex tier.
- The number of users regularly querying Flex tier logs.
- The frequency and types of queries you run. For example, the query time windows you typically use to query your logs.

The number of logs stored in the Flex tier has the largest impact on the size needed to performantly query the data. Datadog recommends the following compute sizes based on log volume:
| Size                                      | Volume (events stored)   |
| ----------------------------------------- | ------------------------ |
| Extra Small (XS)                          | 20 - 50 billion          |
| Small (S)                                 | 50 - 200 billion         |
| Medium (M)                                | 200 - 500 billion        |
| Large (L)                                 | 500 billion - 1 trillion |
| Contact your [Customer Success Manager][7]| 1T+                      |

> **Note**: The recommended number of users is not the number of concurrent queries. However, it is more likely for queries to run concurrently as the number of users increases. Therefore, the concurrency available is higher for queries run on larger compute instances than on smaller instances.

Reviewer: ```suggestion
**Note**: The recommended number of users is not the number of concurrent queries. However, it is more likely for queries to run concurrently as the number of users increases. Therefore, the available concurrency is higher for queries run on larger compute instances than on smaller instances.
```

> Queries that push the limits of the concurrency of a given compute tier slows down the return of the query results and could cause other concurrent queries to slow down as well.

Reviewer: ```suggestion
Queries that push the limits of the concurrency of a compute tier, slow down the return of the query results and can also cause other concurrent queries to slow down.
```

Compute tiers are billed at a flat rate. See the [pricing page][6] for more information.

## Enable and disable Flex Logs

> Flex Logs can be enabled or disabled at the organization level. You must have the `flex_logs_config_write` permission to be able to do that.

Reviewer: ```suggestion
You can enable or disable Flex Logs at the organization level. You must have the `flex_logs_config_write` permission to do so.
```
